src/models/horizontal-scaling-summary/test-data.ts

```diff
@@ -14,6 +14,8 @@ export const testData: TestDependencyDataType<HorizontalScalingSummaryModel> = [
     minReplicas: 3,
     currentCPUUtilizationPercentage: 47,
     targetCPUUtilizationPercentage: 32,
+    currentMemoryUtilizationPercentage: 17,
+    targetMemoryUtilizationPercentage: 33,
   },
   {
```

Comment:
if model is updated, tests needs to be updated
```suggestion
  },
  {
    __testDescription: 'Valid partial object',
    maxReplicas: 2,
    minReplicas: 3,
  },
  {
```